src/components/EventToast/EventToast.js

```diff
@@ -12,7 +12,15 @@ export default function EventToast({ event, id, onClick, t }) {
         </div>
         <MdOutlineClose onClick={onClick} className="cross-icon" color="white" />
       </header>
-      <p className="toast-body">{event.bodyText}</p>
+      {Array.isArray(event.bodyText) ? (
+        event.bodyText.map((s, i) => (
```

Comment:
It would be a little clearer to use readable variables here like 'text', but not critical